pkgs/by-name/se/session-desktop/update.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts
+
+set -euo pipefail
+
+currentVersion="$(nix-instantiate --eval --raw -A session-desktop.version)"
+latestVersion="$(curl -s https://api.github.com/repos/session-foundation/session-desktop/releases/latest | jq -r .tag_name | sed 's/^v//')"
```

Comment:
This needs to optionally take a GITHUB_TOKEN, please see other parts of the code on how to do this.